src/NServiceBus.Core/Endpoint.cs

```diff
@@ -0,0 +1,34 @@
+namespace NServiceBus
+{
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Provides facory methods for creating endpoint instances.
+    /// </summary>
+    public static class Endpoint
+    {
+        /// <summary>
+        /// Creates a new endpoint based on the provided configuration.
+        /// </summary>
+        /// <param name="configuration">Configuration.</param>
+        public static IInitializableEndpoint Create(BusConfiguration configuration)
+        {
+            Guard.AgainstNull(nameof(configuration), configuration);
+            var endpoint = configuration.Build();
+            return endpoint;
+        }
+
+        /// <summary>
+        /// Creates and starts a new endpoint based on the provided configuration.
+        /// </summary>
+        /// <param name="configuration">Configuration.</param>
+        public static async Task<IStoppableEndpoint> CreateAndStartAsync(BusConfiguration configuration)
```

Comment:
the FeatureDev issue calls this method `StartAsync` which I like much more than CreateAndStartAsync (because then it should be CreateAndInitializeAndStartAsync).